content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Changing The Kubernetes Package Repository
 content_type: task
-weight: 120
+weight: 150
```

Comment:
Last because: why didn't you do this [a year ago](https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/) (or more)?